src/ng/log.js

```diff
@@ -11,6 +11,9 @@
  *
  * The main purpose of this service is to simplify debugging and troubleshooting.
  *
+ * To reveal the location of the calls to `$log` in the JavaScript console,
+ * "blackbox" the Angular source in your browser.
```

Comment:
I believe this will be confusing for people that don't know what `blackboxing` means (in this context). I would at least add a link to an explanation of blackboxing.